qt_binder/raw_widgets.py

```diff
@@ -693,7 +693,7 @@ def add_buttons_from_context(self, context):
 ]
 
 
-for obj in vars().values():
```

Comment:
Perhaps the error is due to running the tests in a non-standard fashion?  I get three errors like the following if I run the tests from within the source directory (e.g. via `nosetests *.py` from within the `qt_binder/tests` folder).

However, the errors do not occur if I am outside the source folder and call `python -m nose qt_binder`.  

example error:
```======================================================================
ERROR: Failure: RuntimeError (dictionary changed size during iteration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/lee8rx/anaconda/lib/python3.5/site-packages/nose/failure.py", line 39, in runTest
    raise self.exc_val.with_traceback(self.tb)
  File "/Users/lee8rx/anaconda/lib/python3.5/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/Users/lee8rx/anaconda/lib/python3.5/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/Users/lee8rx/anaconda/lib/python3.5/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/Users/lee8rx/anaconda/lib/python3.5/imp.py", line 234, in load_module
    return load_source(name, filename, file)
  File "/Users/lee8rx/anaconda/lib/python3.5/imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 693, in _load
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 662, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/Users/lee8rx/src/my_git/qt_binder/qt_binder/tests/test_raw_widgets.py", line 19, in <module>
    from ..raw_widgets import BasicGridLayout, GroupBox, HBoxLayout, Label, \
  File "/Users/lee8rx/src/my_git/qt_binder/qt_binder/raw_widgets.py", line 696, in <module>
    for obj in vars().values():
RuntimeError: dictionary changed size during iteration
```